policyengine/simulation.py

```diff
@@ -73,6 +78,9 @@ class Simulation:
     """The baseline tax-benefit simulation."""
     reform_simulation: CountrySimulation | None = None
     """The reform tax-benefit simulation."""
+    data_version: str | None = None
```

Comment:
__nit:__ This applies to other args here, as well
```suggestion
    data_version: Optional[str] = None
```